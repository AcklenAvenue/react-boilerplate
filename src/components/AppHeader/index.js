// @flow
import React from 'react';
import logo from '../../logo.svg';
import {PageHeader} from 'react-bootstrap';

const AppHeader = () => {
  return (
    <PageHeader bsClass="App-header">
      <img src={logo} className="App-logo" alt="logo" />
      Welcome to React
    </PageHeader>
  );
};

export default AppHeader;
