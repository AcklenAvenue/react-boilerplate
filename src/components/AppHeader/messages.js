import { defineMessages } from 'react-intl';

const messages = defineMessages({
  Welcome: {
    id: 'App.Header.Welcome',
    defaultMessage: 'Welcome to React'
  }
});

export default messages;
