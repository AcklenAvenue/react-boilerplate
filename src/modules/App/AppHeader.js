// @flow
import React, {Component} from 'react';
import logo from '../../logo.svg';
import {PageHeader} from 'react-bootstrap';

type Props = {};

class AppHeader extends Component<Props> {
  render() {
    return (
      <PageHeader bsClass="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        Welcome to React
      </PageHeader>
    );
  }
}

export default AppHeader;
