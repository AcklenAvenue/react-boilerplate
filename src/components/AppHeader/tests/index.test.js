import React from 'react';
import AppHeader from '../index';
import renderer from 'react-test-renderer';

describe('<App />', () => {
  it('Expect to be render correctly', () => {
    const wrapper = renderer.create(<AppHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
