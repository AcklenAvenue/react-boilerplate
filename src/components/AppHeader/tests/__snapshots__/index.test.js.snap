// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Expect to be render correctly 1`] = `
<div
  className="sc-bdVaJa kWmPOr page-header"
>
  <h1>
    <img
      alt="logo"
      className="sc-bwzfXH ldyXqg"
      src="logo.svg"
    />
    Welcome to React
  </h1>
</div>
`;
